delete_namespace.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Only allow script to run as
+if [ "$(whoami)" != "root" ]; then
+  echo "This script needs to be run as root. Try again with 'sudo $0'"
+  exit 1
+fi
+
+if [ -z "$NETNS_NAME" ]; then
+  echo Namespace name required, aborting.
+  exit 1
+fi
+
+if [ -z "$ADDR_NET" ]; then
+  echo IP address of namespace required, aborting.
+  exit 1
+fi
+
+# name of the default interface to connect to the Internet
+iface_default=$(route | grep '^default' | grep -o '[^ ]*$')
+
+# name of paired interfaces
+iface_local="$NETNS_NAME-veth0"
+
+# deletes namespace, virtual interfaces associated with it, and iptables rules
+ip netns delete "$NETNS_NAME"
+ip link delete "$iface_local"
+iptables -t nat -D POSTROUTING -s "$ADDR_NET" -o "$iface_default" -j MASQUERADE
+iptables -D FORWARD -i "$iface_default" -o "$iface_local" -j ACCEPT
+iptables -D FORWARD -o "$iface_default" -i "$iface_local" -j ACCEPT
```

Comment:
Might want to review the ordering of the ip netns delete -> ip link delete -> iptables commands.

Seems they're in the wrong order, you would want to delete the iptables rules, delete the ip, then finally the namsepace.

Otherwise you'll find it leads to errors:
Cannot find device "PIA-veth0"
iptables: No chain/target/match by that name.
iptables: Bad rule (does a matching rule exist in that chain?).
iptables: Bad rule (does a matching rule exist in that chain?).

As you're tearing down the namespace anyway I'm not sure it even needs the ip link delete and iptables -D commands